pagopa/io-app
# chore: [IOAPPFD0-202] Update README.md physical devices section

## Short description
This PR updates the README.md with additional steps in order to run the app on a physical iOS device.

## List of changes proposed in this pull request
- Adds a bullet point for the `node` environment flag;
- Adds a bullet point for the CIE testing ;
- Removes the `.xcode.env.local` procedure in the troubleshooting section because it doesn't currently work;
- Clarifies that the bundle identifier change is required only if you're not part of the PagoPA S.p.A. organization;
- Removes the wrong EOL added by #5135;
-  Replaces the deprecated highlight syntax (more on this [here](https://github.com/orgs/community/discussions/16925)).

## How to test
You can check the markdown preview [here](https://github.com/pagopa/io-app/blob/IOAPPFD0-202-update-readme-ios-build/README.md#physical-devices).
